src/components/ObjectArrayEditor.vue

```diff
@@ -15,12 +19,12 @@
     </el-table-column>
     <el-table-column width="100px">
       <template #header>
-        <el-button @click="addItem" size="small">
+        <el-button type="text" @click="addItem">
```

Comment:
type `text` has been deprecated.
type a" has been deprecated.

Reply:
Oh, yep. I found this in element UI doc